api/client/webclient/webclient.go

```diff
@@ -219,13 +219,18 @@ func Ping(cfg *Config) (*PingResponse, error) {
 		return nil, trace.Wrap(err)
 	}
 	defer resp.Body.Close()
-	if resp.StatusCode == http.StatusBadRequest {
-		per := &PingErrorResponse{}
-		if err := json.NewDecoder(resp.Body).Decode(per); err != nil {
-			return nil, trace.Wrap(err)
+
+	if resp.StatusCode != http.StatusOK {
+		slog.DebugContext(req.Context(), "Received unsuccessful ping response", "code", resp.StatusCode)
+		errResp := &PingErrorResponse{}
+
+		if err := json.NewDecoder(resp.Body).Decode(errResp); err != nil {
+			return nil, trace.Wrap(err, "cannot parse unsuccessful ping response")
```

Comment:
Log status code and body here, so we have info to debug?

Reply:
Status is logged a couple of lines above already: `slog.DebugContext(req.Context(), "Received unsuccessful ping response", "code", resp.StatusCode)`.

I can log the body though. I did consider it already, but I didn't like how it elaborate it makes the whole thing. (Even though it actually just adds one extra `io.ReadAll` call with error handling).

Reply:
I think showing the body (under `--debug`) could be very helpful in some instances.